src/easylink/runner.py

```diff
@@ -1,3 +1,13 @@
+"""
+======
+Runner
+======
+
+This module contains the main function for running a pipeline; it is intended to
```

Comment:
I recognize that we probably don't want the API reference to specifically document at the module level, i.e. I should stop referring to "this module" since the title of that section is not necessarily the module's name (e.g. here, the title of the documentation section is "Runner" whereas the module name is "runner.py"). 

I'll clean it up in a future PR. We should either
- lean in to the module level thing, where each doco section is literally named the module name
- pivot to subjects (which just happen to be mostly encapsulated into discrete modules).